hrcase/hrcase.php

```diff
@@ -341,7 +341,10 @@ function hrcase_civicrm_alterContent( &$content, $context, $tplName, &$object )
 }
 
 function hrcase_civicrm_post( $op, $objectName, $objectId, &$objectRef ) {
-  if ($objectName == 'Activity' && isset($objectRef->case_id) && !activityCreatedByTaskandAssignments($objectRef->activity_type_id)) {
+  // PCHR-1177: $objectRef is just an instance of CRM_Activity_DAO_Activity, with values set from parameters sent through request.
+  // Thus, it cannot be used to obtain data for activity that could be in database for given activity id.  We need to load the info from DB first!
+  // Passing activity id instead of object, so data can be loaded from DB within function.
+  if ($objectName == 'Activity' && isset($objectRef->case_id) && !activityCreatedByTaskandAssignments($objectId)) {
```

Comment:
The right place for these type of comments is the commit message and/or the PR description. Comments on the code can easily get outdated and will give the wrong information to the developer